doc/source/whatsnew/v0.18.1.txt

```diff
@@ -74,6 +74,58 @@ API changes
 - ``CParserError`` is now a ``ValueError`` instead of just an ``Exception`` (:issue:`12551`)
 
 
+- A ``CParserError`` is now raised instead of a generic ``Exception`` in ``read_csv`` when the C engine cannot parse a column (:issue:`12551`)
```

Comment:
move all of these to the below subsection as bullet points

just out the pr number once